### PR TITLE
Develop af new variables

### PR DESCRIPTION
**Pull request by carioca-au**

### CODE REVIEW

File: dags/utils/url_request_utils.py [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+#
+#
+# def get_s3_contents_and_attributes(
+#     s3_conn_id: str,
+#     bucket_name: str,
+#     key: str,
+#     params: dict = {},
+#     region_name: str = "us-west-2",
+# ):
+#     """
```
Comment (alexgleith) [inline dags/utils/url_request_utils.py:31]: So much comments.

File: dags/sentinel-1_indexing.py [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+DAG to periodically index Sentinel-1 data.
+
+"""
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.kubernetes.secret import Secret
+from airflow.kubernetes.volume import Volume
+from airflow.kubernetes.volume_mount import VolumeMount
+from airflow.contrib.operators.kubernetes_pod_operator import KubernetesPodOperator
```
Comment (alexgleith) [inline dags/sentinel-1_indexing.py:13]: Why is this all being marked as new?

**Review by alexgleith — COMMENTED** [review]
This is crazy weird... too many changes all at once.

**Comment by carioca-au** [comment]
I'll break it down

**Pull request closed without merging**